perception/object_merger/src/object_association_merger/node.cpp

```diff
@@ -69,9 +72,22 @@ ObjectAssociationMergerNode::ObjectAssociationMergerNode(const rclcpp::NodeOptio
   remove_overlapped_unknown_objects_ =
     declare_parameter<bool>("remove_overlapped_unknown_objects", true);
   overlapped_judge_param_.precision_threshold =
-    declare_parameter<double>("precision_threshold_to_judge_overlapped", 0.4);
+    declare_parameter<double>("precision_threshold_to_judge_overlapped");
   overlapped_judge_param_.recall_threshold =
     declare_parameter<double>("recall_threshold_to_judge_overlapped", 0.5);
+  distance_threshold_list_ = declare_parameter<std::vector<double>>("distance_threshold_list");
+
+  // get distance_threshold_map from distance_threshold_list
+  /** TODO(Shin-kyoto):
+   *  this implementation assumes index of vector shows class_label.
+   *  if param supports map, refactor this code.
+   */
+  int class_label = 0;
+  for (const auto & distance_threshold : distance_threshold_list_) {
+    overlapped_judge_param_.distance_threshold_map.insert(
+      std::make_pair(class_label, distance_threshold));
+    class_label++;
+  }
```

Comment:
How about following code?
```suggestion
  overlapped_judge_param_.distance_threshold_map = convertToClassMap(declare_parameter<std::vector<double>>("distance_threshold_list"));
```
also  add this function.
```
namespace {
  std::map<int, double> convertToClassMap(const std::vector<double> & list) {
int class_label = 0;
  for (const auto & distance_threshold : distance_threshold_list_) {
    overlapped_judge_param_.distance_threshold_map.insert(
      std::make_pair(class_label, distance_threshold));
    class_label++;
  }
}
```

Reply:
As you say, it is better for readability

Reply:
Add function in https://github.com/autowarefoundation/autoware.universe/pull/1902/commits/a057f66fde777e636f65840494a1fda45989df20